src/main/java/com/flightstats/hub/dao/aws/S3BatchManager.java

```diff
@@ -65,8 +65,9 @@ private void setupBatch() {
                     appProperties.getAppEnv());
 
             if (channel.isSingle()) {
-                if (!webhookLeaderState.getServers(channel.getName()).isEmpty()) {
-                    log.debug("turning off batch webhook {}", channel.getDisplayName());
+                WebhookLeaderState.RunningState state = webhookLeaderState.getState(channel.getName());
+                if (!state.isStopped()) {
```

Comment:
Should this be paused? Stopped and Paused are used interchangeably here. You seem to use pause in your comments and stopped in actual code.

Reply:
`paused` is the name of an attribute in the webhook configuration record. `stopped` is any webhook that is not running, whether it should be paused or not.

Reply:
They both have state about last delivered, which items have errored, etc. Paused is just one type of stopped webhook, but a webhook could be stopped for other reasons (a hub node is shutting down and releases its lock, a webhook is in the midst of being deleted, etc.)